src/Storages/StorageFilesystem.cpp

```diff
@@ -0,0 +1,356 @@
+#include <chrono>
+#include <filesystem>
+#include <base/types.h>
+#include <Common/ConcurrentBoundedQueue.h>
+#include <Common/logger_useful.h>
+#include <Interpreters/Context.h>
+#include <Processors/ISource.h>
+#include <Storages/StorageFilesystem.h>
+#include <Storages/StorageFactory.h>
+#include <Storages/checkAndGetLiteralArgument.h>
+
+namespace fs = std::filesystem;
+
+namespace DB
+{
+namespace ErrorCodes
+{
+    extern const int DATABASE_ACCESS_DENIED;
+    extern const int DIRECTORY_DOESNT_EXIST;
+    extern const int NUMBER_OF_ARGUMENTS_DOESNT_MATCH;
+}
+
+String fileTypeToString(fs::file_type type)
```

Comment:
It shouldn't be a string.

Reply:
Leftovers from previous PRs that are not used.